pkg/siteconfig/clusterinstance_test.go

```diff
@@ -13,6 +15,10 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+var (
+	defaultClusterInstanceCondition = ClusterInstanceCondition
```

Comment:
```suggestion
	defaultClusterInstanceCondition = metav1.Condition{Type: "", Status: metav1.ConditionTrue}
```
and then could you please remove the variable from `clusterinstance.go`

Reply:
Okay, updated. Thanks! The proposed change will be in place after submitting a new patch for review.